walkthroughs/week-5-tdd/project/src/main/java/com/google/sps/FindMeetingQuery.java

```diff
@@ -15,9 +15,214 @@
 package com.google.sps;
 
 import java.util.Collection;
+import java.util.Collections;
+import java.util.Comparator;
+import java.util.HashSet;
+import java.util.TreeSet;
+import java.util.Set;
+import java.util.List;
+import java.util.ArrayList;
 
 public final class FindMeetingQuery {
+  /**
+   * Finds the possible TimeRanges where the mandatory attendees can attend
+   * @param events_list sorted list of events in ascending start time order
+   * @param request includes mandatory attendees and duration of meeting
+   * @return collection of possible TimeRanges the meeting can be scheduled in  
```

Comment:
Not a suggestion but I like how you introduced the parameters and return variable! A will-do for me. :)

Reply:
Same here :)